gulpfile.js

```diff
@@ -1,6 +1,7 @@
 var gulp = require('gulp');
 var concat = require('gulp-concat');
 var sass = require('gulp-sass');
+var less = require('gulp-less');
```

Comment:
There's an official sass port of bootstrap that can be installed with bower. `bower install bootstrap-sass`

We could do away with the less requirement using that, right?